ultralytics/engine/exporter.py

```diff
@@ -204,8 +204,8 @@ def __call__(self, model=None) -> str:
             self.args.half = False
             assert not self.args.dynamic, "half=True not compatible with dynamic=True, i.e. use only one."
         self.imgsz = check_imgsz(self.args.imgsz, stride=model.stride, min_dim=2)  # check image size
-        if self.args.int8 and (engine or xml):
```

Comment:
This will remove OpenVINO from export dynamic INT8 models. Is that what we want?

Reply:
This condition causes that when you want to export a model with OpenVINO with `dynamic=False` and `int8=True`, the model is still exported dynamically, because of line `208`, which sets `self.args.dynamic=True`.